### PR TITLE
refactor: make general load model function in utils

### PR DESCRIPTION
**Pull request by sebastian-weisshaar**
The length of the generation of falcon fine-tuned models is wrong. This PR aims to fix this by changing the generation configuration

### CODE REVIEW

File: jerboa/utils/load_model.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                lora_dir = artifact.download(tmpdir)
+                model = load_peft_model(base_model=model, lora_dir=lora_dir)
+        else:
+            model = load_peft_model(base_model=model, lora_dir=lora_dir)
+
+    return model
+
+
+if __name__ == '__main__':
+    model = load_model(lora_dir='jinaai/falcon-7b')
```
Comment (samsja) [inline jerboa/utils/load_model.py:83]: non need for this main entry point, it should rather be a test

File: jerboa/utils/load_model.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        return PeftModel.from_pretrained(
+            model=base_model,
+            model_id=lora_dir,
+        )
+
+    if lora_dir:
+        # Check if the lora weights are from wandb or huggingface
+        # Wandb sources have two forward slashes, huggingface sources have one
+        from_wandb = len(lora_dir.split('/')) == 3
```
Comment (samsja) [inline jerboa/utils/load_model.py:68]: seems a bit hacky, is there a safer way to do this ?

Comment (sebastian-weisshaar) [reply jerboa/utils/load_model.py]: changed

File: tests/test_model_loading.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from jerboa.utils.load_model import load_model
+import pytest
+
+
+@pytest.mark.slow
```
Comment (samsja) [inline tests/test_model_loading.py:5]: you might need to add a marker to the pyproject.toml https://github.com/docarray/docarray/blob/main/pyproject.toml

**Pull request merged**